pkg/pillar/cmd/volumemgr/updatestatusCT.go

```diff
@@ -0,0 +1,333 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package volumemgr
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	uuid "github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
+)
+
+// Returns changed
+// XXX remove "done" boolean return?
+func doUpdateCT(ctx *volumemgrContext, status *types.ContentTreeStatus) (bool, bool) {
+
+	log.Infof("doUpdate(%s) name %s", status.Key(), status.DisplayName)
```

Comment:
Can you correct the function name in the log calls to mach the functio name?

Reply:
Resolved